www-client/uget/metadata.xml

```diff
@@ -12,6 +20,10 @@
 			You'll find the plugin in the app's settings.</flag>
 		<flag name="hide-temp-files">Make temporary files that are used while downloading hidden.</flag>
 		<flag name="gtk3">Use x11-libs/gtk+:3 instead of x11-libs/gtk+:2</flag>
+		<flag name="appindicator">Add support for AppIndicator</flag>
+		<flag name='openssl'>Use <pkg>dev-libs/openssl</pkg> instead of <pkg>net-libs/gnutls</pkg></flag>
+		<flag name='sockets'>Enable JSON-RPC over unix domain socket</flag>
```

Comment:
To be honest, this flag name looks wrong. 'Sockets' make me think of all kinds of sockets, not just AF_UNIX.

Reply:
I committed most of the changes already but this one. Which one do you recommend? I went with sockets because of gpsd but control-socket is used by clsync to enable a similar feature, should that be used instead?

Reply:
What is the purpose of this socket, exactly? (you may also want to include this in description)

Reply:
It enables a JSON-RPC interface, I'm not sure what the use case is (maybe some 3rd party gui?) or how many users actually use this feature.